### PR TITLE
Fix issues with using BigQuery Connection created from UI

### PR DESCRIPTION
**Pull request by kaxil**
This PR creates a constant for `CONN_TYPE_TO_DATABASE` and adds `gcpbigquery` to it. It then uses this for other places .

closes https://github.com/astro-projects/astro/issues/252

### CODE REVIEW

**Comment by kaxil** [comment]
Would love reviews on this @utkarsharma2 @tatiana

File: src/astro/constants.py [hunk 1]
```diff
@@ -36,3 +36,13 @@ class Database(Enum):
 SUPPORTED_DATABASES = list({const.value for const in Database})
 
 UNIQUE_TABLE_NAME_LENGTH = 63
+
```
Comment (utkarsharma2) [inline src/astro/constants.py:39]: can we use aenum package(https://pypi.org/project/aenum/). It will simplify the code.

Comment (utkarsharma2) [reply src/astro/constants.py:39]: TBH I'm not sure about the stability of the project.

Comment (kaxil) [reply src/astro/constants.py:39]: Naah, don't think there is enough value to add an external dependency

**Review by utkarsharma2 — APPROVED** [review]

**Pull request merged**